docs/src/rsh/step/index.md

```diff
@@ -56,7 +57,7 @@ each([Alice, Bob], () => {
   const pretzel = interact.random(); });
 ```
 
-An @{defn("each")} local step statement can be written as `{!rsh} each(PART_TUPLE () => BLOCK)`, where `{!rsh} PART_TUPLE` is a tuple of participants and `{!rsh} BLOCK` is a block.
+An @{defn("each")} local step statement can be written as `{!rsh} each(PART_TUPLE () => BLOCK)`, where `{!rsh} PART_TUPLE` is a <a class="ui-tooltip" title="variables that are used to store multiple items"><span style="cursor: help;">tuple</span></a> of participants and `{!rsh} BLOCK` is a block.
```

Comment:
Tuples are not variables, they are values. You could say "values that store multiple other values"